app/experimenter/nimbus-ui/src/lib/visualization/utils.tsx

```diff
@@ -55,3 +65,29 @@ export const getSortedBranches = (analysis: AnalysisData) => {
   });
   return sortedBranches;
 };
+
+/**
+ * Find the most extreme upper or lower bound value
+ * for an outcome across all branches.
+ *
+ * This is used to scale the confidence interval bars
+ * shown for a metric.
+ */
+export const getExtremeBounds = (
+  sortedBranches: string[],
+  results: AnalysisDataOverall,
+  outcomeSlug: string,
+) => {
+  let extreme = 0;
+  sortedBranches.map((branch) => {
+    const branchComparison = BRANCH_COMPARISON.UPLIFT;
+    const metricDataList =
+      results[branch].branch_data[outcomeSlug][branchComparison]["all"];
+    metricDataList.forEach((dataPoint: FormattedAnalysisPoint) => {
+      const { lower, upper } = dataPoint;
+      const max = Math.max(Math.abs(lower!), Math.abs(upper!));
+      extreme = max > extreme ? max : extreme;
+    });
+  });
+  return extreme;
+};
```

Comment:
It almost feels like this could use a test to make sure it's setting this as expected but I know it's one of our `utils` so it's excluded from coverage... Maybe we could use a test somewhere else to test that `bounds` prop?